src/chkstat.cpp

```diff
@@ -18,25 +18,19 @@
  ****************************************************************
  */
 
-#include <dirent.h>
```

Comment:
`<dirent.h> is just dropped. is it no longer necessary?

Reply:
I'm not sure why this was still there. Maybe `opendir()` was used in the past. But now it's no longer necessary.